client/src/pages/CreateGamePage.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { ChangeEvent, FC, useState } from 'react'
+import { observer } from "mobx-react-lite";
+import { PageWrapper } from "../components/PageWrapper";
+import { GroupBox } from "../components/GroupBox";
+import appStore from "../stores/appStore";
+import styles from "./styles.module.scss";
+import { START_GAME_LABEL } from "../utils/constants";
+import { Button } from "../components/Button";
+import { TextInput } from "../components/TextInput";
+import { CreateGameFormPlaceholders } from "../components/GameDetails/const";
+import { ButtonTitle } from "../components/Button/const";
+import { v4 as uuid } from "uuid";
+import { useGameDetails } from "../hooks/useGameDetails";
+import { Dropdown } from "../components/Dropdown";
+import classes from './createGamePage.module.scss';
+import { AddForm } from "../components/GameDetails/AddForm";
+import { useNavigate } from "react-router-dom";
+
+const options = [
+    { value: 'clicker', label: 'Clicker' },
+    { value: 'keyboard', label: 'Keyboard'},
+];
+
+type Option = {
+    value: string;
+    label: string
+}
+
+export const CreateGamePage: FC = observer(() => {
+    const {
+        setName,
+        changeSettingsGame,
+        setUrlGame,
+        user,
+        gameUrl,
+    } = appStore;
+    const [roomLimit, setRoomLimit] = useState(user.roomLimit);
+    const [gameDuration, setGameDuration] = useState(user.gameDuration);
+    const [nameUser, setNameUser] = useState(user.userName);
+    const [selectedOption, setSelectedOption] = useState<Option | null>(null);
+    const [joinUrl, setJoinUrl] = useState(gameUrl);
+    const roomId = uuid();
+    const { createGame, joinGame } = useGameDetails();
+    const navigate = useNavigate();
+
+    const onChangeRoomLimitHandler = (e: ChangeEvent<HTMLInputElement>) => {
+        const inputValue = e.currentTarget.value;
+        switch(true) {
+            case +inputValue < 2 && inputValue !== '':
+                setRoomLimit(2);
+                break;
+            case +inputValue > 5:
+                setRoomLimit(5);
+                break;
+            default:
+                setRoomLimit(inputValue as unknown as number);
+        }
+    };
+    const onChangeGameDurationHandler = (e: ChangeEvent<HTMLInputElement>) => {
+        const inputValue = e.currentTarget.value;
+        switch(true) {
+            case +inputValue > 60:
+                setGameDuration(60);
+                break;
+            default:
+                setGameDuration(inputValue as unknown as number);
+        }
```

Comment:
I think, it will be better.
```suggestion
        const inputValue = e.currentTarget.value || 0
        setGameDuration(+inputValue > 600 ? 600 : inputValue as unknown as number)
```